docs/products/access/appendix/access-events.md

```diff
@@ -138,9 +138,17 @@
 | AUTHZ REQUEST SCOPE INVALID                                   | One or more of the requested scopes is not available for the specified client.                                                                                                                                       |
 | AUTHZ REQUEST NOT AUTHORIZED FOR SCOPE                        | The user is not authorized for the requested scope. See the [scope verification service](../topics/integration-extension/scope-verification/scope-verification.md) topic guide for more info.                        |
 | AUTHZ REQUEST GRANT CREATED                                   | An authorization code is created for the requested client and scopes.                                                                                                                                                |
+| AUTHZ REQUEST DEVICE GRANT CREATED                            | A device grant is created for the requested client and scopes.                                                                                                                                                       |
```

Comment:
You seem to have added the Events in the "uncategeorized" list, and I would prefer adding a new category for OAUTH Device flow.